composer.json

```diff
@@ -17,7 +17,9 @@
     "monolog/monolog": ">=1.13.1",
     "predis/predis"  : "v0.8.5",
     "domnikl/statsd": ">=1.0.2",
-    "psr/log" : ">=1.0.0"
+    "psr/log" : ">=1.0.0",
+    "guzzlehttp/guzzle" : "^3.8",
+    "firebase/php-jwt": "^3.0"
```

Comment:
It's a client library, so you want to make it as liberal as possible for the application, so I don't think the lock file is appropriate.

But, definitely, you want to avoid breaking changes.